code/game/objects/items/devices/scanners/sequence_scanner.dm

```diff
@@ -70,6 +102,18 @@
 		else
 			to_chat(user, span_notice("[get_display_name(mutation)]"))
 
+/obj/item/sequence_scanner/proc/makeup_scan(mob/living/carbon/target, mob/living/user)
```

Comment:
Usually it's fine not to autodoc small procs like this, but I'd prefer if you did that here since it could be confusing what you're doing here for outsides